sdk/src/types/block/error.rs

```diff
@@ -300,7 +300,7 @@ impl fmt::Display for Error {
             Self::InvalidInputKind(k) => write!(f, "invalid input kind: {k}"),
             Self::InvalidInputCount(count) => write!(f, "invalid input count: {count}"),
             Self::InvalidInputOutputIndex(index) => write!(f, "invalid input or output index: {index}"),
-            Self::InvalidSignedBlockLength(length) => write!(f, "invalid signed block length {length}"),
+            Self::InvalidBlockLength(length) => write!(f, "invalid signed block length {length}"),
```

Comment:
```suggestion
            Self::InvalidBlockLength(length) => write!(f, "invalid block length {length}"),
```

Reply:
I think I found it in the search as well.